EndToEndExample/actions/train/train.py

```diff
@@ -111,7 +111,7 @@ def train(params):
     conn_params = {}
     for p in connection['params']:
         conn_params.update({p['name']: p['value']})
-    print(conn_params)
+    print({k:v for k,v in conn_params.items() if v not in ('secretKey', 'publicKey')})
```

Comment:
The secret was being printed in the pod logs because secrets in a connection are returned as clear text (when in a skill)